src/main/java/org/sagebionetworks/web/client/widget/team/JoinTeamWidget.java

```diff
@@ -79,6 +82,8 @@ public JoinTeamWidget(JoinTeamWidgetView view,
 		this.jsonObjectAdapter = jsonObjectAdapter;
 		this.gwt = gwt;
 		this.wikiPage = wikiPage;
+		this.progressWidget = progressWidget;
```

Comment:
I agree, the progress widget looks fine.  Could you add the css style "modal-fullscreen" to the Modal though (to maximize the width)?